src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -432,6 +432,15 @@
         az acr run -r MyRegistry https://github.com/Azure-Samples/acr-tasks.git -f build-hello-world.yaml --platform linux
 """
 
+helps['acr pack'] = """
+type: command
+short-summary: Queues a task that builds an app and pushes it into an Azure Container Registry.
+examples:
+  - name: Queue a build for the current directory with the default Oryx-based buildpack.
+    text: >
+        az acr pack -r MyRegistry -t {{.Run.Registry}}/node-app:{{.Run.ID}} .
```

Comment:
>{{.Run.Registry}} [](start = 37, length = 17)

Do you require users to always prefix {{.Run.Registry}} in the image name? For `ac acr build` command, we will automatically append the registry info in Tasks runtime. 

I expect pack provide the same experience. If user type the command like the following, you will automatically add the {{,Run.Registry}} prefix.

az acr pack -t node-app:{{.Run.ID}}

Reply:
"we will automatically add the registry to the name" -> where exactly does this happen?

Reply:
Here is the runtime code we prefix the image tag with registry name https://github.com/Azure/acr-builder/blob/9675c7eeda779692ab1ba2c1216cb3e963f08258/util/prefixes.go#L25

Reply:
does this happen on the server side? where is `acr-builder` run?

Reply:
`acr build` use `build` step which does the prefix in server side. You use `cmd` which is a general purpose step. So you need to handle the prefix in client side (CLI).

Reply:
Done. I ignored the case of image names that start with `library/`